lib/backpex/html/resource/resource_index_table.html.heex

```diff
@@ -94,37 +55,24 @@
           <.resource_field name={name} item={item} {assigns} />
         </div>
       </td>
-      <td class="p-0">
-        <div x-bind:style="'width: ' + actionsWidth + 'px !important'"></div>
-      </td>
-      <td
-        x-bind:class="hasOverflow ? 'border-base-100 border-l' : ''"
-        x-bind:style={"hasOverflow ? '#{shadow_sm_left()}' : ''"}
-        class={[
-          "item-action-column text-base-content absolute right-0 px-4 py-0 last:rounded-br-box",
-          index_row_class(assigns, item, selected?(@selected_items, item), index)
-        ]}
-      >
-        <div
-          class="text-base-content/75 flex items-center justify-end space-x-2"
-          x-bind:style="`height: ${rowHeight - 1}px !important; width: ${actionsWidth - 32}px !important`"
-        >
-          <div
+      <td class={[
+        index_row_class(assigns, item, selected?(@selected_items, item), index),
+        sticky_col_class(),
+        "text-base-content/75 !bg-base-100 border-base-200 z-10 w-0 px-6"
+      ]}>
+        <div class={["flex items-center justify-end space-x-2"]}>
+          <button
             :for={{key, action} <- row_item_actions(@item_actions)}
             :if={@live_resource.can?(assigns, key, item)}
-            class="tooltip hover:z-30"
-            data-tip={action.module.label(assigns, item)}
+            type="button"
+            phx-click="item-action"
+            phx-value-action-key={key}
+            phx-value-item-id={Map.get(item, @live_resource.config(:primary_key))}
+            aria-label={action.module.label(assigns, item)}
+            title={action.module.label(assigns, item)}
```

Comment:
@Flo0807 @krns 

I had to remove the item action tooltips because they cannot break out of the `sticky` container without breaking everything. :-(  Can we deal with that? I would be fine with just a `title` attribute for now.

I guess we could find a solution with the Popover API, but that requires an ID for every single tooltip as far as I can see... maybe something for later?

Everything else works pretty good... I got this working with a `position: sticky;` column and a `IntersectionObserver` to dynamically display the column shadow. 

![Bildschirmfoto 2025-02-06 um 00 29 08](https://github.com/user-attachments/assets/2bdab5c1-a734-4dad-8557-a0f086ea291b)

Reply:
I think that's acceptable. But it would be good if we could create an issue for it and add it to the changelog (and link the ticket).